src/main/java/seedu/fast/logic/commands/AppointmentCommand.java

```diff
@@ -44,25 +39,19 @@
             + PREFIX_APPOINTMENT + "2021-10-25 "
             + PREFIX_APPOINTMENT_VENUE + "Ion \n"
             + COMMAND_WORD + " 1 "
-            + PREFIX_APPOINTMENT + "2021-10-25 \n\n"
-            + "Parameters (to delete an appointment): \nINDEX (must be a positive integer), "
-            + PREFIX_DELETE_APPOINTMENT + APPOINTMENT_DELETE_COMMAND + "\n"
-            + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_DELETE_APPOINTMENT + APPOINTMENT_DELETE_COMMAND;
+            + PREFIX_APPOINTMENT + "2021-10-25 \n\n";
 
     public static final String MESSAGE_ADD_APPOINTMENT_SUCCESS = "Added appointment with %1$s: %2$s %3$s %4$s";
-    public static final String MESSAGE_UPDATE_APPOINTMENT_SUCCESS = "Updated appointment with %1$s: %2$s %3$s "
-            + "%4$s";
-    public static final String MESSAGE_DELETE_APPOINTMENT_SUCCESS = "Deleted appointment with %1$s";
-    public static final String MESSAGE_DELETE_APPOINTMENT_FAILED = "No appointment with %1$s yet!";
+    public static final String MESSAGE_ADD_APPOINTMENT_FAILURE = "Appointment already exist! "
+            + "Edit the appointment or delete to re-add!";
```

Comment:
I think changing the message to "The appointment already exists! Use the edit command to change details, or delete it and add it again!" would be clearer.